install/playground/docker/nginx.conf

```diff
@@ -25,7 +64,7 @@ server {
 	   proxy_pass http://meshery-meshery-1:9081;
```

Comment:
Is the uri correct here? It wont be http://meshery:9081 ?

Reply:
Yes, as the meshery container gets named meshery-meshery-1, (i.e. network-container name-serial no.)